<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>998</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XMage Launcher</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/resources.qrc">
    <normaloff>:/icon-mage.png</normaloff>:/icon-mage.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="updateButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>570</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { background-color: #3d3d3d; color: white; border: 1px solid #555; border-radius: 4px; }
QPushButton:hover { background-color: #4d4d4d; }
QPushButton:pressed { background-color: #2d2d2d; }
QPushButton:disabled { background-color: #2a2a2a; color: #666; }</string>
    </property>
    <property name="text">
     <string>Check Updates</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clientButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>570</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { background-color: #3d3d3d; color: white; border: 1px solid #555; border-radius: 4px; }
QPushButton:hover { background-color: #4d4d4d; }
QPushButton:pressed { background-color: #2d2d2d; }
QPushButton:disabled { background-color: #2a2a2a; color: #666; }</string>
    </property>
    <property name="text">
     <string>Launch Client</string>
    </property>
   </widget>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>10</y>
      <width>171</width>
      <height>97</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../resources/resources.qrc">:/label-xmage.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="log">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>140</y>
      <width>601</width>
      <height>371</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: black;
color: white;</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="downloadButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>570</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { background-color: #3d3d3d; color: white; border: 1px solid #555; border-radius: 4px; }
QPushButton:hover { background-color: #4d4d4d; }
QPushButton:pressed { background-color: #2d2d2d; }
QPushButton:disabled { background-color: #2a2a2a; color: #666; }</string>
    </property>
    <property name="text">
     <string>Download XMage</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>520</y>
      <width>601</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="serverButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>570</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { background-color: #3d3d3d; color: white; border: 1px solid #555; border-radius: 4px; }
QPushButton:hover { background-color: #4d4d4d; }
QPushButton:pressed { background-color: #2d2d2d; }
QPushButton:disabled { background-color: #2a2a2a; color: #666; }</string>
    </property>
    <property name="text">
     <string>Launch Server</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clientServerButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>570</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { background-color: #3d3d3d; color: white; border: 1px solid #555; border-radius: 4px; }
QPushButton:hover { background-color: #4d4d4d; }
QPushButton:pressed { background-color: #2d2d2d; }
QPushButton:disabled { background-color: #2a2a2a; color: #666; }</string>
    </property>
    <property name="text">
     <string>Settings</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
